doc/source/release/2.0.0-notes.rst

```diff
@@ -1,45 +1,313 @@
-.. currentmodule:: numpy
-
-==========================
+=========================
 NumPy 2.0.0 Release Notes
-==========================
-
-
-Highlights
-==========
+=========================
 
 
 New functions
 =============
```

Comment:
Indentation.

Reply:
What should be the desired indentation here? 
I see that e.g. `1.23.0-notes.rst` also has `New functions` written this way.

Reply:
Happens, the notes get copied one to another :) Probably doesn't matter, but headings flush on left works.